src/index.js

```diff
@@ -1,42 +1,68 @@
+/* eslint-disable import/no-mutable-exports */
+/* eslint-disable import/prefer-default-export */
 import './style.css';
-import { displayBox } from './check.js';
-
-const allTasks = document.querySelector('.task-list');
-
-const listArray = [
-  {
-    description: 'Complete To-Do list',
-    completed: false,
-    index: 0,
-  },
-  {
-    description: 'Take a walk',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Watch a movie',
-    completed: false,
-    index: 2,
-  },
-];
-
-const showTasks = (tasks) => `
-<li data-id="${tasks.index}">
-<input data-id="${tasks.index}" type="checkbox" name="" class="check-list">
-<label class="list" for="">${tasks.description}<i class="fas fa-ellipsis-v"></i></label>
-</li>
-`;
-allTasks.innerHTML = listArray.map((tasks) => showTasks(tasks)).join('');
-
-displayBox();
-
-// local Storage
-
-const listArraySerialised = JSON.stringify(listArray);
-
-localStorage.setItem('listArray', listArraySerialised);
-
-const listArrayDeserialised = JSON.parse(localStorage.getItem('listArray'));
-
-listArrayDeserialised();
+import { getTodos } from './task.js';
+
+const inputText = document.querySelector('.add');
+const listTask = document.querySelector('.add-task');
+const clear = document.querySelector('.clear-all');
+const todos = JSON.parse(localStorage.getItem('todos')) || [];
+export let editIndex = null;
+
+function saveLocalTodos({ index, description, completed }) {
+  todos.push({ index, description, completed });
+  localStorage.setItem('todos', JSON.stringify(todos));
+  console.log(todos);
+}
+
+const showTasks = (e) => {
+  e.preventDefault();
+  if (editIndex !== null) {
+    // eslint-disable-next-line no-use-before-define
```

Comment:
- [ ] <b>Optional:</b> You overused `eslint-disable`. it's like making changes in Eslint config. they represent style guidelines that you share with your team - which is a group of all Microverse students. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/) and [Zendesk](https://microverse.zendesk.com/hc/en-us/articles/1500002085202-February-2nd-2021-linter-errors-as-a-reason-of-project-invalidation-)). kindy try to fix the errors instead of ignoring them.